stubs/regex/regex/regex.pyi

```diff
@@ -647,6 +647,6 @@ class Match(Generic[AnyStr]):
     @overload
     def __getitem__(self, key: int | str, /) -> AnyStr | Any: ...
     def __copy__(self) -> Self: ...
-    def __deepcopy__(self) -> Self: ...
+    def __deepcopy__(self, /, memo: Unused) -> Self: ...
```

Comment:
it seems like the `memo` argument is also positional-only:

```
 % uv run --with=regex --no-project python 
Installed 1 package in 5ms
Python 3.13.0 (main, Oct  8 2024, 11:56:29) [Clang 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import regex
>>> m = regex.match("foo", "foo")
>>> m
<regex.Match object; span=(0, 3), match='foo'>
>>> m.__deepcopy__(memo=None)
Traceback (most recent call last):
  File "<python-input-8>", line 1, in <module>
    m.__deepcopy__(memo=None)
    ~~~~~~~~~~~~~~^^^^^^^^^^^
TypeError: Match.__deepcopy__() takes no keyword arguments
>>> m.__deepcopy__(None)
<regex.Match object; span=(0, 3), match='foo'>
```

Reply:
I typo'd the location of the /, I meant to make it positional only. Thanks